### ISSUE TITLE
Image preview config does not working with Image.PreviewGroup

### ISSUE THREAD

**Issue by jmatelowski** [ant-design/ant-design#28386]
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/c9e5af93-09f2-421d-a8c1-3bbfeb1416fe-forked-6zsqi?file=/index.js)

### Steps to reproduce
Click on Image.
onVisibleChange method does not invoke, preview container is wrong.

Comment out Image.PreviewGroup wrapper.
Everything works smooth.

### What is expected?
Preview config should work as for single Image.

### What is actually happening?
Preview config is ignored.

| Environment | Info |
|---|---|
| antd | 4.9.3 |
| React | React 16.14.0 |
| System | Macos |
| Browser | Chrome Version 87.0.4280.88  |

---
Moving preview config to PreviewGroup container also does not work.

**Comment by shaodahong** [comment]
yep, `onVisibleChange` is not support if use `Image.PreviewGroup`, maybe we can note in the document

**Comment by jmatelowski** [comment]
Is there any opportunity to support `getContainer` in the future? Or is there any option to manipulate preview component without this prop?

**Comment by shaodahong** [comment]
If customize `getContainer` when use `Image.PreviewGroup`, we may to support it.

**Comment by Sunnyjson** [comment]
Will the Image.PreviewGroup configuration be supported in future releases

**Comment by EmilyyyLiu** [comment]
Thanks for the feedback!

In the latest version of antd, when using Image.PreviewGroup, the preview configuration should be placed on the PreviewGroup component, not on individual Image components.

Correct usage:
```jsx
<Image.PreviewGroup
  preview={{
    onOpenChange: (visible, prev, current) => console.log(visible, prev, current),
    getContainer: "#imageMount"
  }}
>
  <Image src="..." />
  <Image src="..." />
</Image.PreviewGroup>
```

Demo: https://codesandbox.io/p/devbox/xiang-ce-mo-shi-antd-6-5-1-forked-8pcf2y?workspaceId=ws_T98hW5dcqudRAGy3wbJh83

Note: `onVisibleChange` is deprecated. Please use `onOpenChange` instead, which works correctly.

If you have any further questions, feel free to discuss.
